.azure-templates/gradle_job.yml

```diff
@@ -23,3 +23,12 @@ jobs:
       jdkVersionOption: "1.${{ parameters.jdk }}"
       jdkArchitectureOption: 'x64'
       tasks: "${{ parameters.task }}"
+  - task: CopyFiles@2
+  displayName: 'Copy Files to: $(build.artifactStagingDirectory)'
+    inputs:
+      Contents: '**/*.apk'
```

Comment:
The path to the debug build for release is `apks/settings_apk-debug.apk`. Perhaps could `apks/settings_apk-debug.apk` be enough?

`build.artifactStagingDirectory` should be `Build.ArtifactStagingDirectory`?

Reply:
Go test with '**/*.apk'. If that doesn't work, let's try the absolute path.